### ISSUE TITLE
Glyphicons?

### ISSUE THREAD

**Issue by TiggaBiscuit** [jpcormack/www#8]

**Comment by jpcormack** [comment]
Yes! There use to be ones down the bottom of the page. Think I removed them at some point. Font know why.

**Comment by TiggaBiscuit** [comment]
[Icons.pdf](https://github.com/jpcormack/www/files/823034/Icons.pdf)

Maybe these?

**Comment by jpcormack** [comment]
They look good. There's a img folder in the project where we can put those.



Sent from my Samsung Galaxy smartphone.


-------- Original message --------
